.github/workflows/actions.yaml

```diff
@@ -0,0 +1,18 @@
+name: Run Communication Services Tests
+
+om: pull_request
+
+jobs:
+  test:
+  runs-on: ubuntu-latest
+  
+  steps:
+    - uses: actions/checkout@v3
+
+    - name: Run lint and test
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.4.1
+    - run: npm install
```

Comment:
use `yarn` commands instead of `npm`

Reply:
@StefanoPitto can you try this?
also, is this working when you run the `act` command locally?